spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-lattice/src/main/java/org/springframework/cloud/dataflow/module/deployer/lattice/TaskModuleDeployer.java

```diff
@@ -76,7 +81,20 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		for (Map.Entry<String, String> entry : request.getDefinition().getParameters().entrySet()) {
 			environmentVariables.add(new EnvironmentVariable(entry.getKey(), entry.getValue()));
 		}
-
+		// Module launcher properties from Deployment properties
```

Comment:
Does anyone disagree on *not* to support passing module resolver/launcher properties via deployment properties?
It might be ok just to have the support to set the module resolver/launcher properties for the spring-cloud-dataflow-adming at the startup level.